src/gf/meta.py

```diff
@@ -824,6 +824,45 @@ def centroid_position(self):
         return tuple(float(x) for x in
                      (time, lat, lon, north_shift, east_shift, depth))
 
+    def moments(self):
```

Comment:
Every subclass implements moments(). Is it legitimate that this method defaults to None or are there reasons against this?